netneurotools/plotting.py

```diff
@@ -258,37 +258,104 @@ def plot_conte69(data, lhlabel, rhlabel, surf='midthickness',
         Default: (0, 0, 0)
     kwargs : key-value mapping
         Keyword arguments for `mayavi.mlab.triangular_mesh()`
+    Returns
+    -------
+    scene : mayavi.Scene
+        Scene object containing plot
+    """
+
+    return plot_fslr(data, lhlabel, rhlabel, surf_atlas='conte69',
+                     surf_type=surf, vmin=vmin, vmax=vmax, colormap=colormap,
+                     colorbar=colorbar, num_labels=num_labels,
+                     orientation=orientation, colorbartitle=colorbartitle,
+                     backgroundcolor=backgroundcolor,
+                     foregroundcolor=foregroundcolor, **kwargs)
+
+
+def plot_fslr(data, lhlabel, rhlabel, surf_atlas='conte69',
+              surf_type='midthickness', vmin=None, vmax=None,
+              colormap='viridis', colorbar=True, num_labels=4,
+              orientation='horizontal', colorbartitle=None,
+              backgroundcolor=(1, 1, 1), foregroundcolor=(0, 0, 0),
+              **kwargs):
+
+    """
+    Plots surface `data` on a given fsLR32k atlas
+
+    Parameters
+    ----------
+    data : (N,) array_like
+        Surface data for N parcels
+    lhlabel : str
+        Path to .gii file (generic GIFTI file) containing labels to N/2 parcels
+        on the left hemisphere
+    rhlabel : str
+        Path to .gii file (generic GIFTI file) containing labels to N/2 parcels
+        on the right hemisphere
+    surf_atlas: {'conte69', 'yerkes19'}
```

Comment:
```suggestion
    surf_atlas: {'conte69', 'yerkes19'}, optional
        Surface atlas on which to plot `data`
```

Reply:
Sounds good! So I made the changes you suggested! I'll let you merge the pull request (so that you can review the changes) once the checks are completed!